lib/config_module/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ConfigModule
-  VERSION = "1.2.3".freeze
+  VERSION = "1.2.3"
```

Comment:
Undo "Removed redundant freeze on immutable string" until 2.3 is the baseline Ruby version.

Reply:
Fixed in https://github.com/acook/config_module/pull/47/commits/bf3fabb20c4e723a7fc6f0e19d2feb3134867956